storybook/src/docs/form/components/stories/NumberField.stories.tsx

```diff
@@ -0,0 +1,36 @@
+import { FinalForm, NumberField } from "@comet/admin";
+import { storiesOf } from "@storybook/react";
+import * as React from "react";
+
+import { apolloRestStoryDecorator } from "../../../../apollo-rest-story.decorator";
+
+storiesOf("stories/Form/Components", module)
+    .addDecorator(apolloRestStoryDecorator())
+    .add("NumberField", () => {
+        return (
+            <FinalForm mode="add" onSubmit={() => {}}>
+                <NumberField name="number" label="Number" fullWidth />
+            </FinalForm>
+        );
+    });
+
+storiesOf("stories/Form/Components", module)
+    .addDecorator(apolloRestStoryDecorator())
+    .add("WithDecimals", () => {
+        return (
+            <FinalForm mode="add" onSubmit={() => {}}>
+                <NumberField name="number" label="Number with three decimals" decimals={3} fullWidth />
+            </FinalForm>
+        );
+    });
+
+storiesOf("stories/Form/Components", module)
```

Comment:
The stories should be grouped into `NumberField` imo, similar to "Date & Time Pickers", instead of having each story directly under `/Form/Components`.

<img width="264" alt="Screenshot 2024-05-29 at 09 23 16" src="https://github.com/vivid-planet/comet/assets/6264317/e9f51303-5283-4b9c-ba90-7aadf4a71dc0">

```suggestion
storiesOf("stories/Form/Components/NumberField", module)
    .addDecorator(apolloRestStoryDecorator())
    .add("NumberField", () => {
        return (
            <FinalForm mode="add" onSubmit={() => {}}>
                <NumberField name="number" label="Number" fullWidth />
            </FinalForm>
        );
    });

storiesOf("stories/Form/Components/NumberField", module)
    .addDecorator(apolloRestStoryDecorator())
    .add("WithDecimals", () => {
        return (
            <FinalForm mode="add" onSubmit={() => {}}>
                <NumberField name="number" label="Number with three decimals" decimals={3} fullWidth />
            </FinalForm>
        );
    });

storiesOf("stories/Form/Components/NumberField", module)
```